submissions/charlesBanda/html/index.html

```diff
@@ -1,81 +1,175 @@
+
+			</figure>
+	<h2>Product description</h2>
+			<!--product discription-->
+			<p>Winter Gloves for Men and Women -20℉ Cold Weather Warm Thermal
+				Protection Touchscreen& Water Resistant. They have sensitive touchscreen
+				function and allow you to play phone in cold winter without taking out your hands.
+				The elastic cuffs have excellent windproof performance and can protect your whole hands
+				from the cold winds. The water resistant function can keep your hands dry and warm in small
+				snows or rains. They are your must-have winter accessories and can be perfect gift for
+				Thanksgiving Day or Christmas during the cold winter seasons.</p>
+				<ul>
+					<li>Material: warm fleece and faux wool lining</li>
+					<li>Size Guide & 100% Customer Service Warranty—To choose the suitable size,
+						please refer to the size chart image before you make the purchase.
+						We provide 100% customer service, you could contact us anytime via email
+						if you have any problems of the product</li>
+					<li>The Must-have for Winter Activities—These winter touchscreen gloves are unisex
+						design, suitable for both men and women. They are your must-have in cold winter.
+						You could wear them for various outdoor activities or sports: driving, running,
+						walking the dog, snow shoveling, or snowmobiling, etc.</li>
+					<li>Water Resistant & Wind Proof—The back of these cold proof gloves are water resistant,
+						but not entirely waterproof. They could resist small drizzles or snows in cold season.
+						The 4 inch long elastic cuffs cover your wrists, which can block cold winds and rains
+						and give you full protection for your hands</li>
+					<li>Made in Zambia or Imported</li>
+          <br>
+         <h3>Size guide</h3>
+				  <p>To choose the suitable size, please measure your middle finger length and palm size.</p>
+				  <li>S: Middle Finger Length: 3.15inch, Palm Size: 7.47inch</li>
+					<li>M: Middle Finger Length: 3.35inch, Palm Size: 7.87inch</li>
+					<li>L: Middle Finger Length: 3.35inch, Palm Size: 8.27inch</li>
+					<li>XL: Middle Finger Length: 3.55inch, Palm Size: 8.67inch</li>
+			   </ul>
+				 <!-- Similar items details-->
+			<h2>Similar items viewed by others</h2>
+			<ol>
+				<li><a href="#">Riders Fluffy Gloves </a></li>
+					<li><a href="#">Winter Unisex Gloves</a></li>
+						<li><a href="#">Chrismas Gloves</a></li>
+							<li><a href="#">Warm Sports Comfy Gloves</a></li>
+			</ol>
+			<!-- contact details-->
+      <h3>Contact us </h3>
+			<ul>
+					<li><a href="http://www.facebook.com" target="_blank">Facebook</a></li>
+					<li><a href="http://www.twitter.com" target="_blank">Twitter</a></li>
+					<li><a href="http://www.pinprest.com" target="_blank">Pinprest</a></li>
+			</ul>
+</body>
+<!--copyright-->
+<footer><p>&copy; 2020 Msika.com<p></footer>
+</html>
+=======
+<!DOCTYPE html>
```

Comment:
Why are you adding another Html tag here. You should move it into another file or remove it completely